src/components/DraggableItem.module.scss

```diff
@@ -1,27 +1,51 @@
+$dark-degree: 70%;
+$orange: #ff6e5b;
+$orange-dark: darken($orange, $dark-degree);
+$blue: #5fdaf5;
+$blue-dark: darken($blue, $dark-degree);
+$yellow: #f5ef5f;
+$yellow-dark: darken($yellow, $dark-degree);
+
 .draggableItem {
   display: inline-block;
-  color: white;
-  padding: 0.3rem 0.5rem;
+  padding: 0.3rem 0.6rem;
   cursor: grab;
-  box-shadow: 0 0 4px 2px inset rgb(44, 44, 44);
-}
-
-.textOrange {
-  background: rgb(197, 128, 0);
-}
-
-.textPurple {
-  background: rgb(70, 10, 148);
+  border-width: 4px;
+  border-bottom: 7px;
+  border-style: solid;
+  border-radius: 10px;
+  font-size: 30px;
 }
 
-.textPurpleLight {
-  background: rgb(133, 55, 236);
+.orange {
```

Comment:
.orange, .blue, .yellow. SCSS rulesets are really similar, I think it could be simplified by introducing a mixin, which accepts the color as a parameter. And the darker variants could be removed as well since we use them only in those places. 

Mixins: https://sass-lang.com/documentation/at-rules/mixin/#arguments